compiler/modules/bir.sexpr/schema.bal

```diff
@@ -38,8 +38,8 @@ public type TypeMergePred readonly & [Label, Operand];
 
 public type Insn ResultInsn|OperandInsn|Unimpl|
                  CallInsn|CallGenericInsn|TypeCondBranchInsn|BranchInsn|TypeCastInsn|TypeTestInsn|CondBranchInsn|MappingConstructInsn|ListConstructInsn|TypeMergeInsn|ListGetInsn;
-public type CallInsn readonly & ["call", FunctionRef, Result, Operand...];
-public type CallGenericInsn readonly & ["call-generic", FunctionRef, Signature, Result, Operand...];
```

Comment:
Lets not use a boolean in text format. It's harder to make sense when reading. Ether an string union or split the whole type in to two.

Reply:
Fixed by splitting in to `call` and `call-rest-list`